examples/logger/README.md

```diff
@@ -0,0 +1 @@
+# logger-example
```

Comment:
Lets add a good description of what the example does here

Reply:
I added a description. Let me know if the new description is good or if it needs to be more extensive.